src/client/wire_protocol/operations.rs

```diff
@@ -53,6 +53,13 @@ pub enum Message {
         query: BsonDocument,
         return_field_selector: Option<BsonDocument>,
     },
+    OpGetMore {
+        header: Header,
+        // ZERO goes here
```

Comment:
The wire protocol specifies that an empty int32 field goes there in the struct, so I thought it would be useful to use a comment as a placeholder to remind anyone reading the code that there needs to be an empty space there: http://docs.mongodb.org/meta-driver/latest/legacy/mongodb-wire-protocol/#wire-op-get-more